spec/request/appeals_request_spec.rb

```diff
@@ -101,6 +110,15 @@
           expect(response).to match_response_schema('appeals')
         end
       end
+
+      it 'returns a successful camel-inflected response' do
+        VCR.use_cassette('caseflow/appeals_null_eta') do
+          get appeals_endpoint
+          expect(response).to have_http_status(:ok)
+          expect(response.body).to be_a(String)
+          expect(response).to match_camelized_response_schema('appeals')
```

Comment:
Surprised that this test doesn't fail without the key-inflection, I checked the appeals schema and it's already camel'ed.  So actually adding these tests and camelized schemas is not helpful for this spec